data_saver/store.py

```diff
@@ -0,0 +1,26 @@
+import os
+import shutil
+import json
+
+
+def clear_data_at_directory(path):
+    for filename in os.listdir(path):
+        file_path = os.path.join(path, filename)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path)
+        except Exception as e:
```

Comment:
When you raise the base exception, it's an antipattern called "Error hiding" ([link](https://en.wikipedia.org/wiki/Error_hiding))

If you're not going to use this exception in your code above, just don't raise it. If you're going to use it somehow, give this exception a name, then.